client/src/components/runs/logs/forms/ShareWithForm.js

```diff
@@ -137,10 +96,28 @@ export default class ShareWithForm extends React.Component {
       </span>
     );
   };
-
+ sortByOverlap = (str1, str2, query) => {
+   if (str1.toLowerCase().indexOf(query) > str2.toLowerCase().indexOf(query)) {
+     return 1;
+   }
+   if (str2.toLowerCase().indexOf(query) > str1.toLowerCase().indexOf(query)) {
+     return -1;
+   }
+   if (
+     str2.toLowerCase().indexOf(query) === str1.toLowerCase().indexOf(query) &&
+     str1.toLowerCase().indexOf(query) >= 0
+   ) {
+     return str1.toLowerCase() > str2.toLowerCase() ? 1 : -1;
+   }
+   return 0;
+ }
```

Comment:
No need to have this function as a method of `ShareWithForm` instance - put it outside of the class

Reply:
Done!